src/controllers/activegate/reconciler/statefulset/statefulset.go

```diff
@@ -25,7 +25,7 @@ const (
 	tenantSecretVolumeName    = "ag-tenant-secret"
 	authTokenSecretVolumeName = "ag-authtoken-secret"
 
-	annotationCustomPropsHash             = dynatracev1beta1.InternalFlagPrefix + "custom-properties-hash"
+	annotationActiveGateHash              = dynatracev1beta1.InternalFlagPrefix + "activegate-hash"
```

Comment:
since the annotation changed, be aware that existing statefulsets of customers will be recreated